beekeeper-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/generator/utils/CleanupDelayExtractor.java

```diff
@@ -48,7 +48,7 @@ public Duration extractCleanupDelay(ListenerEvent listenerEvent) {
     try {
```

Comment:
Sorry but can we change the logic of this class.
And have something like:
```
if tableCleanupDelay is null
  return defaultValue;
else {
try {
      return Duration.parse(tableCleanupDelay);	      return Duration.parse(tableCleanupDelay);
    } catch (DateTimeParseException | NullPointerException e) {	    } catch (DateTimeParseException | NullPointerException e) {
      log.warn(	      log.info(
          "Overridden delay value '{}' for key '{}' cannot be parsed to a Duration for table '{}.{}'. Using default setting.",	          "Overridden delay value '{}' for key '{}' cannot be parsed to a Duration for table '{}.{}'. Using default setting.",
          tableCleanupDelay, propertyKey, listenerEvent.getDbName(), listenerEvent.getTableName());	          tableCleanupDelay, propertyKey, listenerEvent.getDbName(), listenerEvent.getTableName());
      return defaultValue;
}
```
print warn if we found a value that couldn't be parsed. Would also be good to log the value of default that we are actually setting.